Makefile

```diff
@@ -1,4 +1,4 @@
-SHELL = /bin/bash
+SHELL = /usr/bin/env bash
```

Comment:
Why not just `SHELL = bash`?

Reply:
`bash` would be even more flexible at the expense of having a properly set up `$PATH`. No strong opinion on my side for either, I would be fine with both.

Reply:
A proper `PATH` is still needed for `env` to find `bash`.

Reply:
Correct, let me amend.